README.md

```diff
@@ -0,0 +1,12 @@
+# SettlersOfCatan
+
+What we need:
+
+Board:
+* Hexes
+* Ships
+* Resource Cards
+* Developement Cards
+* Settlements
+* Roads
+* Citys
```

Comment:
We should also have Player and Bank

Reply:
Also, I'm not sure if Resource Cards and Development Cards should be part of Board